luminoth/eval.py

```diff
@@ -18,10 +18,9 @@
 @click.option('--from-global-step', type=int, default=None, help='Consider only checkpoints after this global step')  # noqa
 @click.option('override_params', '--override', '-o', multiple=True, help='Override model config params.')  # noqa
 @click.option('--files-per-class', type=int, default=10, help='How many files per class display in every epoch.')  # noqa
-@click.option('--iou-threshold', type=float, default=0.5, help='IoU threshold to use.')  # noqa
-@click.option('--min-probability', type=float, default=0.5, help='Minimum probability to use.')  # noqa
```

Comment:
Is there no use case or metric that actually uses this thresholding for evaluation?

Reply:
Not really, `min_probability` should never be limited.